src/mpyl/build.py

```diff
@@ -51,59 +52,26 @@ def write_run_plan_as_json():
                     }
                     for project_execution in project_executions
                 ]
-                for stage, project_executions in run_properties.run_plan.items()
+                for stage, project_executions in run_properties.run_plan.full_plan.items()
             }
         )
         run_plan_file = Path(RUN_ARTIFACTS_FOLDER) / "run_plan.json"
         os.makedirs(os.path.dirname(run_plan_file), exist_ok=True)
         with open(run_plan_file, "w", encoding="utf-8") as file:
-            console.print(f"Writing simple JSON run plan to: {run_plan_file}")
+            logger.info(f"Writing simple JSON run plan to: {run_plan_file}")
             json.dump(simple_run_plan, file)
 
     write_run_plan_as_json()
 
-    console.print(f"MPyL log level is set to {run_properties.console.log_level}")
-    branch = obj.repo.get_branch
-    main_branch = obj.repo.main_branch
-    tag = cli_params.tag or run_properties.versioning.tag
-
-    if tag is None:
-        if run_properties.versioning.branch and not obj.repo.get_branch:
-            console.print("Current branch is detached.")
-        else:
-            console.log(
-                Markdown(
-                    f"Branch not specified at `build.versioning.branch` in _{DEFAULT_RUN_PROPERTIES_FILE_NAME}_, "
-                    f"falling back to git: _{obj.repo.get_branch}_"
-                )
-            )
-
-        if branch == main_branch:
-            console.log(f"On main branch ({branch}), cannot determine build status")
-            return
-
-    version = run_properties.versioning
-    revision = version.revision or obj.repo.get_sha
-    base_revision = obj.repo.base_revision
-    if tag:
-        console.print(Markdown(f"**Tag:** `{version.tag}` at `{revision}`. "))
-    else:
-        base_revision_specification = (
-            f"`{main_branch}` at `{base_revision}`"
-            if base_revision
-            else f"not present. Earliest revision: `{obj.repo.root_commit_hex}` (grafted)."
-        )
-        console.print(
-            Markdown(f"**Branch:** `{branch}`. Base {base_revision_specification}. ")
-        )
+    logger.info(f"MPyL log level is set to {run_properties.console.log_level}")
 
     result = RunResult(run_properties=run_properties)
-    if result.has_run_plan_projects:
+    if result.has_run_plan_projects():
         console.print(
```

Comment:
This has to stay a print since the logger can't handle a markdown object

Reply:
There's `mpyl.utilities.logging.try_parse_ansi` for that!